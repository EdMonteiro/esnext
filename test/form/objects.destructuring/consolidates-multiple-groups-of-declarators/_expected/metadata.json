{
  "objects.destructuring": [
    {
      "ids": [
        {
          "type": "Identifier",
          "name": "a",
          "typeAnnotation": null
        },
        {
          "type": "Identifier",
          "name": "b",
          "typeAnnotation": null
        }
      ],
      "inits": [
        {
          "type": "MemberExpression",
          "object": {
            "type": "Identifier",
            "name": "obj",
            "typeAnnotation": null
          },
          "property": {
            "type": "Identifier",
            "name": "a",
            "typeAnnotation": null
          },
          "computed": false
        },
        {
          "type": "MemberExpression",
          "object": {
            "type": "Identifier",
            "name": "obj",
            "typeAnnotation": null
          },
          "property": {
            "type": "Identifier",
            "name": "b",
            "typeAnnotation": null
          },
          "computed": false
        }
      ]
    },
    {
      "ids": [
        {
          "type": "Identifier",
          "name": "c",
          "typeAnnotation": null
        },
        {
          "type": "Identifier",
          "name": "d",
          "typeAnnotation": null
        }
      ],
      "inits": [
        {
          "type": "MemberExpression",
          "object": {
            "type": "Identifier",
            "name": "obj2",
            "typeAnnotation": null
          },
          "property": {
            "type": "Identifier",
            "name": "c",
            "typeAnnotation": null
          },
          "computed": false
        },
        {
          "type": "MemberExpression",
          "object": {
            "type": "Identifier",
            "name": "obj2",
            "typeAnnotation": null
          },
          "property": {
            "type": "Identifier",
            "name": "d",
            "typeAnnotation": null
          },
          "computed": false
        }
      ]
    }
  ]
}