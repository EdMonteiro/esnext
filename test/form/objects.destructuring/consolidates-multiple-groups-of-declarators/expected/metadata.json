{
  "objects.destructuring": [
    {
      "ids": [
        {
          "type": "Identifier",
          "name": "a"
        },
        {
          "type": "Identifier",
          "name": "b"
        }
      ],
      "inits": [
        {
          "type": "MemberExpression",
          "computed": false,
          "object": {
            "type": "Identifier",
            "name": "obj"
          },
          "property": {
            "type": "Identifier",
            "name": "a"
          }
        },
        {
          "type": "MemberExpression",
          "computed": false,
          "object": {
            "type": "Identifier",
            "name": "obj"
          },
          "property": {
            "type": "Identifier",
            "name": "b"
          }
        }
      ]
    },
    {
      "ids": [
        {
          "type": "Identifier",
          "name": "c"
        },
        {
          "type": "Identifier",
          "name": "d"
        }
      ],
      "inits": [
        {
          "type": "MemberExpression",
          "computed": false,
          "object": {
            "type": "Identifier",
            "name": "obj2"
          },
          "property": {
            "type": "Identifier",
            "name": "c"
          }
        },
        {
          "type": "MemberExpression",
          "computed": false,
          "object": {
            "type": "Identifier",
            "name": "obj2"
          },
          "property": {
            "type": "Identifier",
            "name": "d"
          }
        }
      ]
    }
  ]
}